### PR TITLE
finished the merge list method

### PR DESCRIPTION
**Pull request by wosunkwo**

### CODE REVIEW

**Comment by wosunkwo** [comment]
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

File: code401-challenges/src/main/java/code401/challenges/linkedlist/LinkedList.java [hunk 1]
```diff
@@ -172,14 +169,30 @@ public int knthFromEnd(int value){
...
+        if(one.head == null){
+            return two.head;
+        }else if(two.head == null){
+            return one.head;
+        }else{
+            Node oneCurrent = one.head;
+            Node twoCurrent = two.head;
+            LinkedList mergedList = new LinkedList();
+            while(oneCurrent != null || twoCurrent != null){
+                if(oneCurrent == null){
```
Comment (evanslaton) [inline code401-challenges/src/main/java/code401/challenges/linkedlist/LinkedList.java:182]: I don't think this conditional will ever happen. If `oneCurrent == null` then the `while` loop won't run again

File: code401-challenges/src/main/java/code401/challenges/linkedlist/LinkedList.java [hunk 2]
```diff
@@ -172,14 +169,30 @@ public int knthFromEnd(int value){
...
+                    mergedList.append(twoCurrent.value);
+                    twoCurrent = twoCurrent.next;
+                }else if(twoCurrent == null){
```
Comment (evanslaton) [inline code401-challenges/src/main/java/code401/challenges/linkedlist/LinkedList.java:185]: Same as the comment above
